README.md

```diff
@@ -80,34 +80,10 @@ Once installed, you can reference the cloud.azure_ops collection content by its
 * [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
 
 
-## Contributing to this collection
+## Contributing
 
-We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against this collection repository.
+See [CONTRIBUTING](./CONTRIBUTING.md) for more info
 
-### Testing and Development
-
-The project uses `ansible-lint` and `black`.
-Assuming this repository is checked out in the proper structure,
-e.g. `collections_root/ansible_collections/cloud/azure_ops/`, run:
-
-```shell
-  tox -e linters
-```
-
-Sanity and unit tests are run as normal:
-
-```shell
-  ansible-test sanity
-```
-
-There are also integration tests in the `molecule` directory which are meant to be run against an azure subscription.
-
-```shell
-    az login
-    molecule test
-```
-
-This collection is tested using GitHub Actions. To know more about CI, refer to [CI.md](CI.md).
```

Comment:
I guess we should include the information regarding the CI testing.

Reply:
Yes, you right, I included this line back in the README